### ISSUE TITLE
Injection moulded visor frames

### ISSUE THREAD

**Issue by drakard** [DoESLiverpool/covid19#92]
@ajlennon I have pricing back from my suppliers. For the simplest option tool cost starts at around roughly £4,000 +VAT with a 3 week lead time. This is for a Prussa compatible design and done entirely in the UK. Please let me know if there is interest in pursuing this option.

**Comment by ajlennon** [comment]
Much better than the 20K I've seen quoted. There is a _lot_ of activity on this nationally and it's hard to know how things are going to play out. I definitely think we all should be trying to agree on a single injection moulded design if possible. I don't see how DoES could support this kind of outlay though.

Also

> Kevin Nicholas-CB10 (Saffon Walden)  9:04 AM
> Hi Alex Solid Solutions (SolidWorks) are talking about an injection moulded version of the headbands comming on line very soon. I had this from a collegue who recieved an email from Solid Solutions I am a member but elected not to recieve thier marketing (maybe a mistake in this instance)
> I have looked at their site but whilst it has loads of info on how to get involved it has no mention of moulding. I will endeavour to investigate this but if this is so it will be fantastic. I know there are companies that specalise in rapid production of injection moulded parts such as ProtoLabs. https://www.protolabs.co.uk/services/injection-moulding/plastic-injection-moulding/
> I should have thought of this sooner.
> If the bulk can be injection moulded we can turn our printers to other parts that need producing?
> protolabs.co.ukprotolabs.co.uk
> Plastic Injection Moulding, Rapid Prototyping | Protolabs
> Our plastic injection moulding process produces custom - rapid prototypes & end-use production parts in 15 days or less. We use aluminium moulds that offer cost-efficient tooling & accelerated manufacturing cycles, and stock more than 100 different thermoplastic resins. Request a free quote today.(88 kB)
> https://www.protolabs.co.uk/media/1011732/900x400-im-service-pim-materials-blue-clear.jpg
> 9:06
> Obviously the design for Moulded parts is different to the printed parts If Solid Solutions have already done the transitional design then it can only be for the good.
> 
> Alex Lennon - Regional Coordinator - North West UK - DoES Liverpool  9:40 AM
> Hi Kev! Yeah there are a few different injection moulded options
> 9:40
> The main problem as I understand it is the cost of creating a tool
> 9:41
> 3DCrowd know they can't fulfil their over 500k orders 3D printing so have been talking to a couple of companies about injection moulding
> 9:41
> I agree it's the way forward
> 
> Kevin Nicholas-CB10 (Saffon Walden)  9:42 AM
> I will find out It looks like Solid Solutions may have sorted this out. (they should do they have enough cash to do this I know ProtoLabs would respond to such a request especialy from the likes of SoliWorks.
> 9:43
> Its not prohibitive in cost but the molds have a finite life. but that is in the hundreds if not thousands.
> 
> Alex Lennon - Regional Coordinator - North West UK - DoES Liverpool  9:43 AM
> Be interesting to see where it goes :smile:
> 9:43
> That's what we need!
> 
> Kevin Nicholas-CB10 (Saffon Walden)  9:43 AM
> Yes a fully functional injection moulded part would be ITRO £20,000
> 
> Alex Lennon - Regional Coordinator - North West UK - DoES Liverpool  9:44 AM
> Yeah
> 9:44
> I have a company up here in Warrington
> 9:44
> I went to get injection moulded prototypes off them last week
> 
> Kevin Nicholas-CB10 (Saffon Walden)  9:44 AM
> proto Labs are quoting in hundreds and with a 14 day turn around.

**Comment by drakard** [comment]
@ajlennon I have not had a good experience with protolabs in the past but maybe they are offering better prices now. Where is the quoted discussion from? Would you be able to let me know where I could follow up regarding injection moulding (or even introduce me if that's appropriate)? I have been given very good unit prices also.

**Comment by ajlennon** [comment]
@drakard can you sign up at 3DCrowd.uk and register on the slack space if not already and I'll introduce - covid-19volunteersuk.slack.com

**Comment by drakard** [comment]
@ajlennon I am registered on the slack space under the name David. I posted today in the Injection Moulding channel, but I can take that down if needed.

**Comment by drakard** [comment]
A company called 3P Innovation https://www.3pinnovation.com/ I have spoken with has an injection mould already under manufacture in the UK. They are also already working towards CE approval.

**Comment by drakard** [comment]
A company called Vetech http://www.vetech.co.uk/ are producing injection moulded frames and can apparently make a mould tool in 2 days "Lead times from as little as 1 day." ! https://www.buxtonadvertiser.co.uk/business/buxton-engineering-firm-makes-nearly-30000-health-worker-masks-after-designing-new-moulding-tool-2537968

**Comment by ajlennon** [comment]
Yeah @goatchurchprime believes there's no good reason for long lead times if the right pressure is applied

**Comment by drakard** [comment]
I just spoke to Vetech and they will be sending us out a box of 120 free sample frames tomorrow. They are a modified Prusa design and should be hole-compatible.

**Comment by ajlennon** [comment]
> I just spoke to Vetech and they will be sending us out a box of 120 free sample frames tomorrow. They are a modified Prusa design and should be hole-compatible.

:clap:

**Comment by JackiePease** [comment]
Just spotted this on Instagram. Liverpool based so may be worth getting in touch: https://www.instagram.com/p/B_PjSNNHmu0/?igshid=2xfctx8geppg

**Comment by drakard** [comment]
> Just spotted this on Instagram. Liverpool based so may be worth getting in touch: https://www.instagram.com/p/B_PjSNNHmu0/?igshid=2xfctx8geppg

Daniel Isler (I believe the person operating this account) created these files: https://grabcad.com/library/prusa-face-shield-headband-injection-moulding-1 . This is his website d2productdevelopment.co.uk .

I have sent him an email to introduce us and to see if he has gone beyond these design files regarding injection moulding so far.

**Comment by drakard** [comment]
Vetech have shared with me their production details and are happy for them to be published. I am considering setting up a static website to list the information I am finding since there seems to be significant disorganisation. In the mean time, I will store the details here (if that's ok).

Supplier: Vetech https://vetech.co.uk
Product: Visor frames ; modified Prusa design
Unit cost: 35p+VAT ex works
Tool/batch cost: None
Lead time: Ready
Capacity: 1650/day
Buyer requirement: None

**Comment by drakard** [comment]
A box of injection moulded visor frames should be arriving at the workspace today. If anyone sees them and you could let me know they arrived (and if they look good / work) that would be super helpful.

**Comment by drakard** [comment]
I have a contact who is able to supply plastic for injection moulding free of charge in appropriate situations.

**Comment by JackiePease** [comment]
Big box of injection moulded visors have arrived for @drakard. They're practically the same form factor as the Prusas, so still have elastic, so more difficult to assemble and clean.
They don't have the sloping part attached to the post for the elastic, which is good.
Very smooth finish, comfortable and shield clips on easily.

**Comment by JackiePease** [comment]
![IMG_20200423_144942_Bokeh](https://user-images.githubusercontent.com/12802256/80108353-1b5cf980-8574-11ea-94b6-e9cbdb391eb4.jpg)
![IMG_20200423_144816_Bokeh](https://user-images.githubusercontent.com/12802256/80108392-26178e80-8574-11ea-8644-fa8b30dcff31.jpg)
![IMG_20200423_144806_Bokeh](https://user-images.githubusercontent.com/12802256/80108412-2e6fc980-8574-11ea-9cde-92108186e9fc.jpg)

**Comment by drakard** [comment]
The injection moulded frames appear to be excellent, although their design requires an elastic strap which is not desirable. They also do not come with the bottom brace for the visor shield. So, currently these will need to be 3D printed. These parts are small so they can be 3D printed quickly, but it would be ideal if they were also moulded. The price is comparable with laser cut frames so that is not currently an advantage. With die cut frames these may be comparatively more expensive. However, injection moulding does have the option to reduce costs further with additional investment. These frames are close to the standardised Prusa design which may be advantageous for adoption, although they inherit the drawbacks of that design. They are more rigid than laser or die cut frames and thus are compatible with a wider variety of visor shield materials.

I will try to identify an alternative to the elastic strap to make this a more suitable option.

**Comment by drakard** [comment]
We should send 50 visors made from these donated samples to end users as quickly as possible to give a good result for the supplier. I believe we are only held up waiting on elastic strapping for this. We will also need to print bottom braces. I believe we should save the remaining frames for now to test with other fastening and visor shield options.

**Comment by drakard** [comment]
I have made a laser cut band prototype which can be used with the injection moulded frames instead of elastic #98

**Comment by drakard** [comment]
I have sliced the bottom reinforcement models required to complete Prusa visors, suitable for our 3d printers.

[Bottom reinforcesments x10 sliced.zip](https://github.com/DoESLiverpool/covid19/files/4565088/Bottom.reinforcesments.x10.sliced.zip)

**Comment by drakard** [comment]
@JackiePease @ajlennon  Could you please let me know, are any of the 3d crowd / prusa project people (ie. particularly Mick) in today? Could I get any contact details for them please?

**Comment by drakard** [comment]
The injection moulded visor sets are now ready for delivery and use.

**Comment by JackiePease** [comment]
I've given the visors to @SeanTomlinson30 to test

**Comment by drakard** [comment]
@JackiePease Thank you! I hope they are liked.

**Comment by drakard** [comment]
As best I understand the visors were sent out. I will be happy to provide any details or continue with this if there is any demand. I have not heard any update from the demand side since they were sent out so I am closing it as I assume there is not a current need.

**Comment by SeanTomlinson30** [comment]
Hello,

They have been sent out however due to loads of administrative issues to the facility receiving them, they are only now being tested. I will have updates soon, I'm promised.

**Comment by drakard** [comment]
That's great to hear. If you need any input from me just let me know.
